src/httpledstripe_esp.cpp

```diff
@@ -1583,7 +1588,8 @@ void deleteConfigFile(void)
 
 void updateConfigFile(void)
 {
-  DynamicJsonDocument doc(8192);
```

Comment:
in the current implementation, the 8192 are sufficent. maybe we stick to that to not stress the dynamic memory too much.